resume_takedata.py

```diff
@@ -9,6 +10,8 @@ def user_detail(self):
 			data = raw_input()
 			user_data.append(data)
 		resume_writedata.Writer().write_data(user_data)
+		write_csv.CreateCsv().csv_create(individual_detail,user_data)
+

```

Comment:
@Nithinkrishna1 User should be able to select from a list of available formats, to which format the program should write to